.github/workflows/code-linter.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: ['1.22']
+        go-version: ['1.23']
```

Comment:
Since we are only running against a single version we might as well set this directly in the setup go step and remove the entire `matrix` block